### PR TITLE
Update installation and correct some words

### PR DESCRIPTION
**Pull request by Emma-Song**
1. 环境名称统一改为 arctern_env
2. 修正单词拼写：RESTful, Python, pip

### CODE REVIEW

File: doc-cn/source/restful/webserver_installation_config.md [hunk 1]
```diff
@@ -81,50 +81,53 @@ cd gui/server/arctern_server
...
 pip install arctern_server
 ```
 
 ## 配置后台 Arctern-Spark 信息
 
-### 配置基于源码安装的 Arctern Restful Server
+### 配置基于源码安装的 Arctern RESTful Server
 
-修改 Arctern 项目中 `gui/server/arctern_server` 路径下的 `config.ini` 文件，配置 Arctern Restful Server 所使用的 Arctern-Spark 后台信息。文件配置示例如下，其中 `spark_master_ip` 和 `port` 分别为后台 Arctern-Spark 中 master 节点的 IP 地址和端口号：
+在 Arctern 项目的 `gui/server/arctern_env` 目录下，修改 `config.ini` 文件以配置 Arctern RESTful Server 所使用的 Arctern-Spark 后台信息。文件配置示例如下，其中 `spark_master_ip` 和 `port` 分别为后台 Arctern-Spark 中 master 节点的 IP 地址和端口号：
```
Comment (yxm1536) [inline doc-cn/source/restful/webserver_installation_config.md:96]: "arctern_server" here is a file directory name hence should not be replaced.

Comment (yxm1536) [reply doc-cn/source/restful/webserver_installation_config.md:96]: The same issue exits in branch0.1.x

File: doc-cn/source/restful/webserver_installation_config.md [hunk 2]
```diff
@@ -147,30 +150,29 @@ master-addr = spark://spark_master_ip:port
...
 ```
 
-## 启动 Arctern Restful Server
+## 启动 Arctern RESTful Server
 
-### 启动基于源码安装的 Arctern Restful Server
+### 启动基于源码安装的 Arctern RESTful Server
 
-在 Arctern 项目的 `gui/server/arctern_server` 目录下使用以下命令启动服务，其中`/path/to/server` 为 Arctern 项目下 `gui/server` 目录的绝对路径。
+在 Arctern 项目的 `gui/server/arctern_env` 目录下执行以下命令启动服务，其中 `/path/to/server` 为 Arctern 项目下 `gui/server` 目录的绝对路径。
```
Comment (yxm1536) [inline doc-cn/source/restful/webserver_installation_config.md:157]: "arctern_server" here is a file directory name hence should not be replaced.
The same issue exits in branch0.1.x

File: doc-cn/source/spark/installation_and_deployment/install_arctern_on_spark_cn.md [hunk 3]
```diff
@@ -107,17 +96,17 @@ Type "help", "copyright", "credits" or "license" for more information.
...
 ```
 
 在文件 `conf/spark-env.sh` 的最后添加以下内容。其中 `[path/to/your/conda]` 为Conda的安装路径。
 
 ```bash
-export PYSPARK_PYTHON=[path/to/your/conda]/envs/arctern_spark/bin/python
+export PYSPARK_PYTHON=[path/to/your/conda]/envs/arctern_env/bin/python
 ```
 
 ### 确认路径配置是否成功
```
Comment (yxm1536) [inline doc-cn/source/spark/installation_and_deployment/install_arctern_on_spark_cn.md:112]: In line 120 typo: 一下

Comment (yxm1536) [reply doc-cn/source/spark/installation_and_deployment/install_arctern_on_spark_cn.md:112]: branch 0.1.x may have the same issue.

File: doc-cn/source/spark/installation_and_deployment/offline_install_arctern_on_spark_cn.md [hunk 4]
```diff
@@ -31,50 +32,49 @@ $ ./install_packages.sh gl      # 安装gl-mesa库
...
 
-  `conda activate arctern-spark`
+  `conda activate arctern_env`
 
 
-> **注意：后续工作必须在 conda 虚拟环境 (arctern-spark) 中进行**
+> 注意：后续工作必须在 conda 虚拟环境 (arctern_env) 中进行。
 
 
 ## 安装 Arctern-Spark
```
[the remark this answers is not in the document]
Comment (Emma-Song) [reply doc-cn/source/spark/installation_and_deployment/offline_install_arctern_on_spark_cn.md:80]: the 1st 'arctern_spark' (in red background) is in the old version, the new version is 'arctern_env' (in green background)

**Pull request merged**